app/controllers/providers/emails_controller.rb

```diff
@@ -3,12 +3,12 @@ class EmailsController < ApplicationController
     include ApplicationDependable
     include Steppable
 
-    def show
-      @form = Applicants::EmailForm.new(model: applicant)
+    def edit
```

Comment:
I don't think you should change this one. In routes this is defined via `resource :email ...` - not the singular. That gives a path ending `/email` for the show view. If you change it to `edit` the path becomes `/email/edit`.

Following this afternoon's meeting, I wonder if we should be using more routes defined by `resource` (singular) and then use `show` rather than `edit` as the default landing page for the controllers.